README.md

```diff
@@ -94,6 +94,7 @@ render(
   math: none,
   h1-level: 1,
   raw-typst: true,
+  table: true,
```

Comment:
Does this need an option? I think it should be fine to just enable it by default.

Reply:
I made it a option with the though that the user can use other preprocessors after cmarker to handle markdown tables according to there requirement if the need arises.